ILRepack/MappingHandler.cs

```diff
@@ -27,13 +27,8 @@ public override int GetHashCode()
                 return scope.GetHashCode() + name.GetHashCode();
             }
 
-            public override bool Equals(object obj)
```

Comment:
According to IEquatable doc, they say we should also override Equals to get consistent behaviour (https://msdn.microsoft.com/en-us/library/ms131187%28v=vs.110%29.aspx)